### ISSUE TITLE
User function namespaces

### ISSUE THREAD

**Issue by sirex** [atviriduomenys/spinta#62]
User functions could have namespaces as suggested in https://gitlab.com/atviriduomenys/spinta/issues/8.

Configuration would look like this:

```python
config = {
    'ufuncs': {
        '': [
            'spinta',
            'vpt',
        ],
        'spinta': [
            'spinta.commands.ufuncs',
        ],
        'vpt': [
            'lodam.ufuncs.vpt',
        ],
    },
}
```

Slight modification would be, that `config['ufuncs']['']` would have a list of other ufunc namespaces, that should be added into global namespaces.

Then function resolution would work as follows:

1. Check if there is a namespace and use it.

2. Check if there is a function in `spinta` namespace.

3. Check if there is a function in `vpt` namespace.

Namespaces can overshadow functions in global namespace and functions at the top of `config['ufuncs']['']` list can overshadow  functions from namespaces lower in the list.

If a function is oveshadowed by a namespace, you can always access it using full path for example `@spinta.spinta()` or `@vpt.eq()`.

So basically it is always possible to access any function from any namespace and control what goes into global namespace.

Namespaces must be unique.

Example:

```
/reports?status.@vpt.eq(final)
```

Here, `status` would resolve to reports column, then `.@vpt` would resolve to a namespace `FuncNs(Bind('status'))`, then `.eq()` would resolve to `vpt.eq(Bind('status'), Bind('final'))` function.

Function namespaces should be nested.
